tests/test_sitemap.py

```diff
@@ -76,4 +68,4 @@ def test_sitemap_cache(self):
 
         ext_key = get_cache_key(page3)
         page3.delete()
-        self.assertEqual(cache.get(ext_key), None)
+        self.assertEqual(cache.get(ext_key), None)
```

Comment:
Please fix this no new line at end of file

Reply:
I believe flake8 would have given an error if this was necessary.

Reply:
Now, Do I need to update that!

Reply:
It should have failed on this to be fair. Add a new line being as you will be making another change.